index.html

```diff
@@ -1,48 +1,285 @@
-            <li><a href="#"><img src="images/github.svg" id="github"></a></li>
-            <li><a href="#"><img src="images/linkedin.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/Hand.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/twitter.svg" alt=""></a></li>
-            <li><a href="#"><img src="images/M.svg" alt=""></a></li>
-        </ul>
-        
-      </section>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul>
+          <li>
+            <a href="#">css</a>
+          </li>
+          <li>
+            <a href="#">html</a>
+          </li>
+          <li>
+            <a href="#">Bootstraps</a>
+          </li>
+          <li>
+            <a href="#">Ruby</a>
+          </li>
+        </ul>
+        <button>see project</button>
+      </div>
+    </section>
+    <section class="cards">
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard.
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">rubby</a>
+          </li>
+        </ul>
+        <button class="card-btn">see project</button>
+      </div>
+      <div class="card-body">
+        <h1 class="card-title">Profesional Art Printing Data</h1>
+        <p class="supporting-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul class="card-ul">
+          <li class="card-li">
+            <a href="" class="card-a">html</a>
+          </li>
+
+          <li class="card-li">
+            <a href="" class="card-a">bootstrap</a>
+          </li>
+

```

Comment:
- [ ] Good job on the changes you have made, but there is still room for improvement if you want your project to be completed. Please use the correct path to display the card and this section because it is empty and only one section is showing. Please fix this for project to be completed :point_down: 
![empty](https://user-images.githubusercontent.com/88894525/199560110-b53642dd-d037-4575-9d51-63130aa977b9.png)